### PR TITLE
Cl abap datfm

### PR DESCRIPTION
**Pull request by dominikpanzer**
Hi,

I wrote a quite naive implementation of the method `CONV_DATE_EXT_TO_INT`, because _really_ correct date validation is painful.  It's the only method of this class that I personally need. The conversion is only possible for dates in format DD.MM.YYYY and is guarded by a regex. 

Greetings,
Dominik

### CODE REVIEW

File: src/date_time/cl_abap_datfm.clas.abap [hunk 1]
```diff
@@ -10,26 +10,25 @@ CLASS cl_abap_datfm DEFINITION PUBLIC.
       RAISING
         cx_abap_datfm.
 
-    CLASS-METHODS get_date_format_des
```
Comment (larshp) [inline src/date_time/cl_abap_datfm.clas.abap:13]: keep this method definition in the source code

**Comment by larshp** [comment]
naive: no problem, one step at a time. Its good to merge after adding the old method definition back in

**Review by larshp — CHANGES_REQUESTED** [review]
see above

**Comment by dominikpanzer** [comment]
done :-)

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**